packages/web-components/src/components/va-sidenav/va-sidenav-item.scss

```diff
@@ -0,0 +1,50 @@
+@import '../../mixins/focusable.css';
+
+:host {
+  display: block;
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+  border-bottom: 1px solid var(--vads-color-base-lightest);
+  &:active {
+    background-color: var(--vads-color-action-focus-on-light);
+    color: var(--vads-color-base);
+    box-shadow: inset 0px -4px 0px var(--vads-color-base);  
+    z-index: 1;
+  }
+}
+
+a {
+  color: var(--vads-color-link);
+  display: block;
+  line-height: var(--vads-font-line-height-4);
+  padding: 0.5rem 0.25rem 0.5rem 1rem;
+  padding-left: var(--va-sidenav-item-link-padding-left, 1rem);
+  text-decoration: none;
+}
+
+[href]:focus {
+  outline: none!important;
```

Comment:
It made me sad to use `!important` but I couldn't find any other way to suppress the (overly) aggressive global focus style coming from the CSS-Library. It was causing a double border.

**Before**
<img width="648" height="235" alt="before" src="https://github.com/user-attachments/assets/890d3c39-b8ba-45f8-9458-2fc58e5636cb" />


**After**

<img width="628" height="248" alt="after" src="https://github.com/user-attachments/assets/4e6fba40-0eb5-4ac7-a1fb-5d369dcc26a3" />